### ISSUE TITLE
simclasses depends on clsim (Trac #2208)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2200]
This breaks the standalone IceRec build.

Migrated from https://code.icecube.wisc.edu/ticket/2208
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:23", 
    "description": "This breaks the standalone IceRec build.", 
    "reporter": "jbraun", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067323910946", 
    "component": "combo core", 
    "summary": "simclasses depends on clsim", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2018-11-27T15:30:58", 
    "milestone": "", 
    "owner": "amedina", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20181127T15:35:21: claudio.kopper commented:

> removed the dependency in r167111 (by commenting it out). If we need this, we should just move the whole clsim/shadow/I3ExtraGeometry* stuff to simclasses.
>

**Comment by icecube-trac** [comment]
Trac update at 20181127T15:35:29: 
* olivas changed priority from "critical" to "blocker"
* olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20181130T19:14:48: 
* olivas commented:

> Andres is working on this now...


* olivas changed owner from "olivas" to "amedina"
* olivas changed status from "accepted" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20181130T20:41:25: 
* amedina changed status from "assigned" to "closed"
* amedina commented:

> In changeset 167201:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167201"
> fixes #2200
> ```


* amedina changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T03:16:59: nwhitehorn commented:

> In changeset 170511:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170511"
> fixes #2200
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:23: 
* olivas commented:

> Milestone Winter Solstice 2018 deleted


* olivas changed milestone from "Winter Solstice 2018" to ""
